.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: check-toml
+        exclude: "^examples/.template/"
+      - id: check-yaml
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/PyCQA/isort
+    rev: 12cc5fbd67eebf92eb2213b03c07b138ae1fb448  # For split_on_trailing_comma.
+    hooks:
+      - id: isort
+        additional_dependencies: [toml]
+        args: ["--settings", "setup.cfg"]  # https://github.com/PyCQA/isort/issues/1872
+        exclude: "^(attic|nursery|examples/.template)/"
```

Comment:
This was a huge pain to get working. Without the `--settings` argument, isort would read setup.cfg when run via `pre-commit run`, but not when run by the actual commit hook. But when I added the `--settings` argument, it started ignoring the `skip` setting in the configuration file, so I replaced it with an `exclude` setting here. It's more convenient to have all the exclusions in the same file anyway.